gradle/dependencies.gradle

```diff
@@ -112,11 +114,16 @@ ext {
       androidXAnnotation        : "androidx.annotation:annotation:${version.androidXAnnotationVersion}",
       androidXAppCompat         : "androidx.appcompat:appcompat:${version.androidXAppCompatVersion}",
       androidXCore              : "androidx.core:core:${version.androidXCoreVersion}",
+      androidXCoreKtx           : "androidx.core:core-ktx:${version.androidXCoreVersion}",
+      androidXLifecycleCommon   : "androidx.lifecycle:lifecycle-common-java8:${version.androidXLifecycle}",
```

Comment:
pretty sure you can remove this when you're using lifecycle `2.4.0` instead of `2.3.1`.

https://developer.android.com/jetpack/androidx/releases/lifecycle#version_24_2